### PR TITLE
Update fetch.js

### PR DESCRIPTION
**Pull request by ccwalkerjm**
The program wasn't close properly.

### CODE REVIEW

**Comment by deitch** [comment]
Overall, this looks good. Nice catch on those errors. A few questions:

1. Can you bump the version patch number for this in `package.json`?
2. We already have a test that does multiple `FETCH` in a row, see [here](https://github.com/deitch/imapper/blob/master/tests/fetch.js#L167) and [here](https://github.com/deitch/imapper/blob/master/tests/fetch.js#L203) . Why did those pass?

**Comment by deitch** [comment]
The tests fail.

```
$ npm test

> imapper@0.7.0 test /Users/adeitcher/Documents/Development/JavaScript/imapper
> grunt

Running "jshint:all" (jshint) task
>> 52 files lint free.

Running "nodeunit:all" (nodeunit) task
Testing 00-master.js.OK
Testing append.js.F.F
>> Normal login - Append flags
>> Error: false == true
>> at Object.<anonymous> (tests/append.js:63:18)
>> at Socket.<anonymous> (lib/mock-client.js:50:17)
>> at emitOne (events.js:115:13)
>> at Socket.emit (events.js:210:7)
>> at TCP._handle.close [as _onclose] (net.js:547:12)

>> Normal login - Append full
>> Error: false == true
>> at Object.<anonymous> (tests/append.js:110:18)
>> at Socket.<anonymous> (lib/mock-client.js:50:17)
>> at emitOne (events.js:115:13)
>> at Socket.emit (events.js:210:7)
>> at TCP._handle.close [as _onclose] (net.js:547:12)

Testing auth-plain.js.......OK
Testing copy.js.OK
Testing create-special-use.js..OK
Testing create.js...OK
Testing delete.js..OK
Testing examine.js..OK
Testing expunge.js.OK
Testing fetch.js.F................
>> imapper tests - FETCH FLAGS
>> Error: false == true
>> at Object.<anonymous> (tests/fetch.js:161:18)
>> at Socket.<anonymous> (lib/mock-client.js:50:17)
>> at emitOne (events.js:115:13)
>> at Socket.emit (events.js:210:7)
>> at TCP._handle.close [as _onclose] (net.js:547:12)

Testing general.js...OK
Testing list.js.....OK
Testing literalplus.js...OK
Testing login.js.....OK
Testing rename.js.OK
Testing search-query.js...........................................OK
Testing search.js.....................................OK
Testing select.js..OK
Testing special-use.js........OK
Testing status.js..OK
Testing storage-plugin.js.............................OK
Testing store.js..........OK
Testing subscribe.js..OK
Testing uid_copy.js..OK
Testing uid_fetch.js.OK
Testing uid_search.js.........................................OK
Testing uid_store.js..........OK
Testing unselect.js..OK
Testing users-plugin.js....OK
Testing x-gm-ext-1.js.....OK
Testing xoauth2.js....OK
Warning: 3/563 assertions failed (1244ms) Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.
```

**Comment by deitch** [comment]
I am losing track. Please comment here when it is ready to merge.

File: lib/server.js [hunk 1]
```diff
@@ -40,8 +40,8 @@ function IMAPServer(options) {
 
     this.options = options || {};
     this.options.credentials = this.options.credentials || {
-        key: fs.readFileSync(__dirname + "/../cert/server.key"),
-        cert: fs.readFileSync(__dirname + "/../cert/server.crt")
+        key: fs.readFileSync(__dirname + "/../../../certs/server.key"),
+        cert: fs.readFileSync(__dirname + "/../../../certs/server.crt")
```
Comment (deitch) [inline lib/server.js]: Why change the certs dir?

Comment (ccwalkerjm) [reply lib/server.js]: ignore this. sorry.  I will revert..sorry.....

**Comment by ccwalkerjm** [comment]
No longer necessary

**Pull request closed without merging**